extensions/virtuallistview/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reactxp-virtuallistview",
-  "version": "0.1.9",
+  "version": "0.1.9-sym-1.0",
```

Comment:
Please don't change the package version. I'll increment it if/when I publish a new official version.

Reply:
This was more done for internal purposes. We have our own internal repository.